policyengine_us/variables/gov/states/id/tax/income/credits/grocery/id_grocery_credit_base.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class id_grocery_credit_base(Variable):
+    value_type = float
+    entity = Person
+    label = "Idaho base grocery credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.id.tax.income.credits.grocery.amount
```

Comment:
again this includes dependents, and this is a person-level field so you don't need to sum to the tax unit. it's a flat amount per person
```suggestion
        return parameters(period).gov.states.id.tax.income.credits.grocery.amount.base
```